kytos/cli/commands/napps/api.py

```diff
@@ -9,7 +9,7 @@
 
 from kytos.utils.napps import NAppsManager
 
-log = logging.getLogger(__name__)
+LOG = logging.getLogger(__name__)
```

Comment:
I'm not sure but I think there was a reason to ignore this linter and keep it lowercase. @cemsbr ?

Reply:
We kept lowercase the ``log`` that the NApp dev imports and uses, in kytos project.